storage/client/client.go

```diff
@@ -1603,6 +1603,7 @@ func (c *StorageClient) RuntimeEVMNFTs(ctx context.Context, limit *uint64, offse
 			&ownerAddrData,
 			&nft.MetadataUri,
 			&metadataAccessedN,
+			&nft.Metadata,
```

Comment:
and then this does unmarshal the JSON from the db into Metadata interface{} as like maps and stuff :shrug: and our API layer later re-marshals it :repeat: